server/routes/bookVisit/cannotBookController.ts

```diff
@@ -9,10 +8,6 @@ export default class CannotBookController {
       const { prisoner } = req.session.bookingJourney
       delete req.session.bookingJourney
 
-      if (prisoner.availableVos >= 1) {
```

Comment:
Removing this check as the data could be stale and it will be re-fetched if user goes back to home page.